server/routes/visitJourney/checkYourBooking.ts

```diff
@@ -68,6 +69,28 @@ export default class CheckYourBooking {
         res.locals.appInsightsOperationId,
       )
 
+      if (isUpdate) {
+        const outcome: OutcomeDto = {
+          outcomeStatus: 'SUPERSEDED_CANCELLATION',
+          text: `Superseded by ${visitSessionData.visitReference}`,
```

Comment:
Not a requirement on the ticket, but figured it made sense to note the new visit reference in the old visit's 'cancellation reason' text so it's recorded against the old visit and the user audit log.